### PR TITLE
Add roadmap guidelines

### PR DESCRIPTION
**Pull request by dreispt**
Implementation proposal for #102

### CODE REVIEW

**Comment by dreispt** [comment]
@yvaucher can you disable that tag automation?

**Comment by yvaucher** [comment]
@dreispt disabled

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -686,3 +686,50 @@ The differences include:
...
+
+    Summary with the functional benefit expected for the users.
+
+    Supporters
+    ----------
+
+    List of people supporting the feature implementation.
+    This might be for work on functional specification, code development,
+    tests and validation, documentation, coordination, or just moral support.
+    The point is that features is no supporters should be removed from the raadmap.
```
Comment (max3903) [inline CONTRIBUTING.md]: The point is that features without supporters

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -686,3 +686,50 @@ The differences include:
...
+
+
+    Functional Specification
+    ========================
+
+    Functional description of the features to implement.
+    May be adapted to user documentation once implemented.
+    Include links to relevant resources.
```
Comment (max3903) [inline CONTRIBUTING.md]: Functional spec should provide the README.rst content (description, configuration, usage).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -686,3 +686,50 @@ The differences include:
...
+
+    Technical Specification
+    =======================
+
+    Technical breakdown of the components and tasks to implement.
+    Include links to relevant resources.
+
```
Comment (max3903) [inline CONTRIBUTING.md]: Technical spec should provide the installation part of the README.rst content (external libraries).

**Comment by gurneyalex** [comment]
@dreispt can you rebase?

I'm :+1: once @max3903's remarks are taken into account.

**Comment by dreispt** [comment]
I came to the conclusion that "roadmap milestones" (ex: `8.0-roadmap` milestone) can be an overkill. Maybe we should stick to plain version milestones for now...

**Comment by max3903** [comment]
@dreispt Shall we close it?

**Pull request closed without merging**